src/app/main/component/events/components/images-container/images-container.component.ts

```diff
@@ -38,7 +38,7 @@ export class ImagesContainerComponent implements OnInit {
   @Output() deleteImagesOutput = new EventEmitter<Array<string>>();
   @Output() oldImagesOutput = new EventEmitter<Array<string>>();
 
-  constructor(private localStorageService: LocalStorageService, private snackBar: MatSnackBarComponent) {}
+  constructor(private events: EventsService, private localStorageService: LocalStorageService, private snackBar: MatSnackBarComponent) {}
```

Comment:
it is better to avoid naming the service 'Events' as this term is often used for system-level constructs, which could lead to confusion